cmd/cloud.go

```diff
@@ -51,19 +53,24 @@ const (
 )
 
 //nolint:gochecknoglobals
-var exitOnRunning = os.Getenv("K6_EXIT_ON_RUNNING") != ""
+var (
+	exitOnRunning   = os.Getenv("K6_EXIT_ON_RUNNING") != ""
+	disableTailLogs = os.Getenv("K6_DISABLE_TAIL_CLOUD_LOGS") != ""
```

Comment:
What do you think about renaming this to `K6_CLOUD_LOGS` with a default value of "1"/true, and disabling it with `K6_CLOUD_LOGS=0`? Having "disable" as part of the name is counter-intuitive to me.

Reply:
I really didn't want to deal with parsing env variables and cli flags ... in here. I guess you also want the cli flag to be renamed? 
I am really not against this ... it's just that because of the configuration mess this is I prefer not to make anything more complex than .. checking that something is defined, which arguably isn't great. 

I will wait for @na-- review so I do a batch of changes

Reply:
`K6_DISABLE_TAIL_CLOUD_LOGS` is not a good option name, for multiple _different_ reasons. `TAIL` and `tailing` will probably be very confusing to Windows users, and `DISABLE` breaks the convention we've had with negative option names like `K6_NO_USAGE_REPORT`, `K6_NO_CONNECTION_REUSE`, etc.

Moreover, negative options are confusing, since they force them to deal with double negatives, like `K6_DISABLE_TAIL_CLOUD_LOGS=false` is stupid. So, :+1: :+1: for `K6_CLOUD_LOGS` or something like that

Reply:
I wonder whether to not disable this by default though .... While it is likely to be harder to find this way ... arguably a lot of people (using CIs for example) wouldn't really want this feature enabled by default ?

Reply:
That's a separate discussion, though yes, I am also not sure if logs should be enabled by default for the `k6 cloud` command :man_shrugging:

Reply:
My problem with `K6_CLOUD_LOGS` is that this sounds like something that should be in the ... output ... also ... disabling this ... doesn't disable the cloud logs ..., it disables(or enables) the tailing of the clouds logs, the cloud logs will still be send and saved (if any)

Reply:
I am tentatively of the opinion that `k6 cloud` logs should be enabled by default. This will make `k6 cloud` feel much more similar to `k6 run`, which is good. And it'd be easy to turn off the logs, but a lot of k6 users might not even discover we have cloud logs if we disable them by default - not everyone is reading the k6 release notes...

Reply:
> My problem with K6_CLOUD_LOGS is that this sounds like something that should be in the ... output ... also ... disabling this ... doesn't disable the cloud logs ..., it disables(or enables) the tailing of the clouds logs, the cloud logs will still be send and saved (if any)

Not sure if I completely understand all your objections ("..." is usually placed at the end of a sentence :wink:), but maybe something like `K6_SHOW_CLOUD_LOGS` then? "Show" is much more understandable than "tail" for non-*nix users and it will more accurately depict what's happening, while also not having the `K6_CLOUD` prefix?